src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -293,6 +292,9 @@
         <Link>Common\Interop\OSX\Interop.SearchPath.cs</Link>
       </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsBrowser)' == 'true'">
+	<Compile Include="$(BclSourcesRoot)\System\Threading\TimerQueue.Browser.cs" />
```

Comment:
I'm not sure about the location. Do we have to have this in shared part SPC for easier discoverability otherwise it should match existing naming rules with the name of `TimerQueue.Browser.Mono.cs`

@jkotas @stephentoub thoughts?